### PR TITLE
Portfolio: Contact Form

### PR DESCRIPTION
**Pull request by Yosefgeda**
### Contact Form
- Added form to contact possible clients.
- Added footer part with links to all social media.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -158,6 +158,28 @@ <h1 class="about-head">About me</h1>
...
+  <footer>
+    <ul class="footer-icons">
+      <li><a href="https://github.com/Yosefgeda"><img src="icons/GitHub.png" alt="github logo"></a></li>
+      <li><a href="www.linkedin.com/in/yosefgeda"><img src="icons/Linkedin.png" alt="linkedin logo"></a></li>
+      <li><img src="icons/Angelist.png" alt="angel loist"></li>
+      <li><a href="twitter.com/yosefgeda"><img src="icons/Twitter.png" alt="twitter logo"></a></li>
+      <li><img src="icons/Medium.png" alt="BigM"></li>
+    </ul>
+    <span class="footer-line"><hr></span>
+  </footer>
```
Comment (bobb-Rob) [inline index.html]: - [ ] You should try to stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding). You got all this well except the spacing, Your contact form lack proper margins on the left and right sides. There should be no scroll bar on the horizontal side of the browser.
- [ ] Every other thing you did well. Well done 👏 

**_Figma Template_**
image

**_Your work_**
image

File: index.html [hunk 2]
```diff
@@ -158,6 +158,28 @@ <h1 class="about-head">About me</h1>
...
     </div>
   </section>
+
+  <section class="contact-section">
+    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    <form action="https://formspree.io/f/mknagrvq" method="POST" class="contact-form">
+      <input type="text" name="name" maxlength="30" placeholder="Full name" class="form-input" required />
+      <input type="email" name="email" placeholder="Email address" class="form-input"required />
+      <textarea maxlength="500" name="description" placeholder="Enter text here" class="form-input form-textarea" rows="5"required ></textarea>
+      <button type="submit" class="submit-btn">Get in touch</button>
```
Comment (bobb-Rob) [inline index.html:168]: - [ ] Please add the hover effects to the button. See your Figma design for the button states.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Yosefgeda , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
Hi @Yosefgeda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**